config/vufind/searches.ini

```diff
@@ -742,3 +742,20 @@ searchAccessPermission = access.api.Search
 
 ; This is the maximum number of results that can be returned in a single response:
 maxLimit = 100
+
+; This section provides settings for optional caching of search requests. Supported
+; for Solr only.
+[SearchCache]
+; Supported adapters: Memcached, Filesystem. Others may also work, but have not been
+; tested.
+;adapter = Memcached
+; Memcached servers (optional, uses localhost:11211 by default)
+;servers[] = "localhost:11211"
```

Comment:
Sorry for the last-minute nitpick, but would it make sense to call this memcached_servers to reduce potential ambiguity? Or can this be addressed directly through the options setting below? (I'm guessing not due to the amount of nesting, but I'd love to be wrong about that). I realize that we agreed earlier that \VuFind\Cache\Manager doesn't have a role here, but I do wonder if it would make sense to have a shared method that translates a VuFind Config object into a cache object in a consistent way, so the settings in the [Cache] section of config.ini could match up with the new settings in [SearchCache] to a greater extent. (I'm not necessarily asking you to do that now -- just brainstorming... but if nothing else, maybe we should open a JIRA ticket about cache consolidation/reorganization).

Reply:
So, you _were_ just as wrong as I was, and the configuration has now been simplified. :)

I added https://vufind.org/jira/browse/VUFIND-1467. Feel free to modify as necessary.

Reply:
Ahh, fantastic, thanks on both counts!